source/stylesheets/_tab-secrets-env_vars.scss.erb

```diff
@@ -122,6 +128,20 @@
 						}
 					}
 
+					.is-expand {
+						@include order(1);
```

Comment:
Order just changes the elements' order *visually*. But e. g. tabbing order is not changed. There should be a better solution.

Reply:
I just wanted to change the order visually :D